packages/backend/src/controllers/v2/ProjectController.ts

```diff
@@ -94,7 +94,6 @@ export class V2ProjectController extends BaseController {
                 .runPaginatedMetricQuery({
                     ...commonArgs,
                     metricQuery,
-                    csvLimit: body.query.csvLimit,
```

Comment:
The query endpoint is not used to download CSVs anymore. We now download CSVs via the `/downloadCsv` endpoints.